docs/app-guide/add-a-onepassword-secret.rst

```diff
@@ -0,0 +1,116 @@
+###########################################
+Add a secret with 1Password and VaultSecret
+###########################################
+
+Whenever possible, static secrets for applications are stored in a 1Password vault before being automatically synced to the Vault service itself and ultimately to Kubernetes ``Secret`` resources via :ref:`vault-secrets-operator`.
+For an overview of secrets architecture, see :ref:`secrets`.
+
+This page provides steps for adding an application secret through 1Password.
+
+Part 1. Open the 1Password vault
+================================
+
+In one password, access the **LSST IT** 1Password team and open the vault named ``RSP-Vault``.
+Items in this vault are synced into Kubernetes ``Secret`` resources.
+
+Part 2. Create a Secret Note
+============================
+
+Each item in a Kubernetes ``Secret`` corresponds to the contents of a secret note in 1Password.
+(Many secret notes can combined into a single Kubernetes ``Secret`` by configuring the ``VaultSecret``).
+
+- The title of the secret note should be formatted as:
+
+  .. code-block:: text
+
+     {{service}} {{env}} {{secret name}}
```

Comment:
I'm not sure anything follows this convention? For instance, your semaphore secrets don't have the secret name in the title; they use the human-readable phrase "GitHub App private key", which I prefer.